srcpkgs/json-c/template

```diff
@@ -1,21 +1,30 @@
 # Template build file for 'json-c'.
```

Comment:
please change to the appropiate header

`# Template file for 'json-c'`

Reply:
Even the dot at the end?